packages/web-app-user-management/src/views/Groups.vue

```diff
@@ -240,62 +208,46 @@ export default defineComponent({
 
   methods: {
     ...mapActions(['showMessage']),
+
     calculateListHeaderPosition() {
-      this.listHeaderPosition = this.$refs.appBar.getBoundingClientRect().height
+      this.listHeaderPosition = this.$refs.template.$refs.appBar.getBoundingClientRect().height
```

Comment:
Soft-fails in unit tests because `this.$refs.template.$refs` is undefined in the unit tests of the view (-> gives an ugly error message in the unit tests)